skorzeli-Buffalo.md

```diff
@@ -0,0 +1,3 @@
+# BUFFALO ROCKS!
+
+But if you are in Dearborn, go to Millers!
```

Comment:
The title of this section is confusingly misleading, given the body content, and may lead to unnecessary travel expenses and time wasted by people seeking Miller's in Buffalo